packages/react/src/components/SuiteHeader/util/uiresources.js

```diff
@@ -42,9 +43,10 @@ const getUiResourcesData = async ({
   const surveyIdParam = surveyId ? `&surveyId=${surveyId}` : '';
   const appIdParam = appId ? `&appId=${appId}` : '';
   const workspaceIdParam = workspaceId ? `&workspaceId=${workspaceId}` : '';
+  const cacheParam = useCache ? `&cache=${true}` : '';
```

Comment:
When build a URL dynamically you must use encodeURIComponent() so that you can't embed malicious content inside of surveyId, appId and workspaceId (cache is actually OK, since, it's not using a variable for the right hand side)

Reply:
Done!